woocommerce/payment-gateway/Handlers/Capture.php

```diff
@@ -0,0 +1,424 @@
+<?php
+/**
+ * WooCommerce Payment Gateway Framework
+ *
+ * This source file is subject to the GNU General Public License v3.0
+ * that is bundled with this package in the file license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://www.gnu.org/licenses/gpl-3.0.html
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@skyverge.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade the plugin to newer
+ * versions in the future. If you wish to customize the plugin for your
+ * needs please refer to http://www.skyverge.com
+ *
+ * @package   SkyVerge/WooCommerce/Payment-Gateway/Admin
+ * @author    SkyVerge
+ * @copyright Copyright (c) 2013-2018, SkyVerge, Inc.
+ * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0
+ */
+
+namespace SkyVerge\WooCommerce\PluginFramework\v5_2_2\Payment_Gateway\Handlers;
```

Comment:
is this supposed to be tagged `5.3.0` as the methods point out in their `since` tags and v5.2.2 is released already? or is this for 5.2.3?

Reply:
I'm keeping the namespace change for last when we actually tag this release, so plugins can more easily test these things without having to change their namespace too

Reply:
cool